kythe/rust/extractor/extractor_test.bzl

```diff
@@ -61,13 +63,13 @@ rust_extractor_test = rule(
         "src": attr.label(
             mandatory = True,
             executable = True,
-            cfg = "host",
+            cfg = "target",
```

Comment:
These should both be `host` no? They are going to be run on the host, not built for the target platform.

Reply:
```
Otherwise, executables that are used at runtime (e.g. as part of a test) should be built for the target configuration. In this case, specify cfg="target" in the attribute.
```
https://docs.bazel.build/versions/main/skylark/rules.html#configurations